lottie/src/main/java/com/airbnb/lottie/utils/Logger.java

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.lottie.utils;
+
+/**
+ * Singleton object for logging. If you want to provide a custom logger implementation,
+ * implements LottieLogger interface in a custom class and replace Logger.instance
+ */
+public class Logger {
```

Comment:
I think you can combine Logger and LottieLogger. Just annotate the static logging functions with `@RestrictTo`

Reply:
Do you mean removing the interface and let people subclass Logger instead?

Reply:
Actually, nevermind, you're correct here. You can leave it as-is.